docs/content/direct/core-chart.md

```diff
@@ -0,0 +1,279 @@
+# KubeStellar Core chart usage
+
+This documents explains how to use KubeStellar Core chart to deploy a new instance of KubeStellar
+with a choice of user-defined KubeFlex Control Planes (CPs).
+
+The information provided is specific for the following release:
+
+```shell
+export KUBESTELLAR_VERSION=0.23.0-alpha.2
+```
+
+## Pre-requisites
+
+To install the Helm chart the only requirement is [Helm](https://helm.sh/).
+However, additional executables may be required to create/manage the cluster(s) (_e.g._, Kind and kubectl),
+to join Workload Execution Clusters (WEKs) (_e.g._, clusteradm),
+and to interact with Control Planes (_e.g._, kubectl), _etc_.
+For such purpose, a full list of executable that may be required can be fund [here](./pre-reqs.md).
+
+The setup of KubStellar via the Core chart requires the existence of a KubeFlex hosting cluster.
+
+This can be:
+
+1. A local **Kind** or **k3s** cluster with an ingress with SSL passthrough and a mapping to host port 9443
+
+This option is particularly useful for first time users or users that would like to have a local deployment.
+
+It is important to note that, when the hosting cluster was created by **kind** or **k3s** and its Ingress domain name is left to default to localtest.me, then its Control Plane name must be also be referenced during the Helm chart installation by setting `--set "kubeflex-controller.hostContainer=<control-plane-container-name>"`.
+The `<control-plane-container-name>` is the name of the container in which kind or k3d is running the relevant control plane. One may use `docker ps` to find the `<control-plane-container-name>`.
+
+If a host port number different from the expected 9443 is used for the Kind cluster, then the same port number must be specified during the chart installation by adding the following argument `--set "kubeflex-controller.externalPort=<port>"`.
+
+By default the KubeStelalr Core chart uses a test domanin `localtest.me`, which is ok for testing on a single host machine. However, scenarios that span more than one machine, it may be useful to set `--set "kubeflex-controller.domanin=<domain>"` to a more appropriate `<domain>` that can be reached from Workload Execution CLusters (WECs).

```

Comment:
s/domanin/domain/

Can be done in follow-on.

Reply:
ok